### PR TITLE
Add insurance (BEP20)

### PR DESCRIPTION
**Pull request by Mazyar60**
Adding info for BEP20 token 'insurance'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

**Comment by Mazyar60** [comment]
Hi
Please update INSURANCE token in Trust wallet

File: blockchains/smartchain/assets/0x64E4fea6e4F3637025c7Bcd878E2B238B01f7D4e/info.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        },
+        {
+            "name": "blog",
+            "url": "https://Insurance.game/blog"
+        }
+    ],
+    "tags": [
+        "Defi"
+    ]
+}
```
Comment (Mazyar60) [inline blockchains/smartchain/assets/0x64E4fea6e4F3637025c7Bcd878E2B238B01f7D4e/info.json:36]: Gamefi

**Comment by cryptomanz1** [comment]
• What is the purpose of this token? How will it benefit Trust Wallet users?
 • How is the token being distributed? Are you conducting an airdrop?
 • Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
 • Is the token listed on a centralized/decentralized exchange?
 • Is a Smart Contract audit completed? Where can we find this information?

**Comment by Mazyar60** [comment]
> • What is the purpose of this token? How will it benefit Trust Wallet users?
>  • How is the token being distributed? Are you conducting an airdrop?
>  • Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
>  • Is the token listed on a centralized/decentralized exchange?
>  • Is a Smart Contract audit completed? Where can we find this information?


Purpose and Benefits to Trust Wallet Users: INSURANCE Token offers decentralized insurance for physical and digital assets, using blockchain technology to provide secure, efficient, and cost-effective services. Trust Wallet users can manage insured digital assets directly from their wallets.

Distribution and Airdrop: The token is being distributed through presales and airdrop campaigns to early adopters, incentivizing growth.

ICO/Presale: Yes, the token was sold in a presale phase.

Exchange Listings: INSURANCE Token is listed on centralized and decentralized exchanges, including Pancakeswap.finance , Azbit.com ,tokpie.com ,vindax.com

Smart Contract Audit: A completed audit can be accessed via the project’s website, providing transparency and security assurances for users.For more details, visit insurance.game.

**Comment by Mazyar60** [comment]
> ### PR Summary
> Files OK: 2
> Token in PR: **BEP20** 0x64E4fea6e4F3637025c7Bcd878E2B238B01f7D4e
> Symbol: **INSURANCE** decimals: 18
> Links: [Logo](https://raw.githubusercontent.com/Mazyar60/assets/br1023-020446/blockchains/smartchain/assets/0x64E4fea6e4F3637025c7Bcd878E2B238B01f7D4e/logo.png) [Info](https://raw.githubusercontent.com/Mazyar60/assets/br1023-020446/blockchains/smartchain/assets/0x64E4fea6e4F3637025c7Bcd878E2B238B01f7D4e/info.json) [Website](https://Insurance.game) [Explorer](https://bscscan.com/token/0x64E4fea6e4F3637025c7Bcd878E2B238B01f7D4e)
> Tags: defi
> Token check error: 
> ❌ invalid value 'https://twitter.com/insurance0game' for twitter link url, allowed only with prefix: https://x.com/
> ❌ number of holders not checked: please, check it manually
> 
> 
> 
> <span style="padding: 5px; background-color: rgb(32,32,32);"></span>
> 
> 
> 
> 

https://x.com/insurance0game

**Comment by Mazyar60** [comment]
Hi
Thank you for your support
Insurance logo is wrong and name not search in add new token
Please check

**Comment by Mazyar60** [comment]
Hello
Why insurance token , haven't logo in trust?

**Comment by cryptomanz1** [comment]
We require smart contract audit from reputable companies. Please see https://developer.trustwallet.com/listing-new-assets/requirements

**Pull request merged**